application/src/test/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBServiceTest.java

```diff
@@ -61,6 +61,7 @@
 import uk.nhs.hee.tis.revalidation.dto.TraineeRequestDto;
 import uk.nhs.hee.tis.revalidation.dto.TraineeSummaryDto;
 import uk.nhs.hee.tis.revalidation.entity.DoctorsForDB;
+import uk.nhs.hee.tis.revalidation.entity.RecommendationGmcOutcome;
```

Comment:
Very good point, I've had a look through and it looks like we store the value of the enum for gmcStatus instead of the enum itself (like we do for tisStatus) 🤦.

So in this case I think the FE will be sending us the value back because that's what it was originally given and we'd have to convert it to and back from the enum anyway.

I would fix this as part of the ticket but I think it might be a big job because it would involve redoing the CDC and manual sync, do you think making another ticket is suitable here?

Reply:
Yes, I think it could be another one for tech debt.